Library/Formula/qemu.rb

```diff
@@ -31,11 +32,27 @@ def install
       --disable-bsd-user
       --disable-guest-agent
     ]
+
     args << (build.with?("sdl") ? "--enable-sdl" : "--disable-sdl")
     args << (build.with?("vde") ? "--enable-vde" : "--disable-vde")
     args << (build.with?("gtk+") ? "--enable-gtk" : "--disable-gtk")
-    ENV["LIBTOOL"] = "glibtool"
+    args << (build.with?("libssh2") ? "--enable-libssh2" : "--disable-libssh2")
+
     system "./configure", *args
     system "make", "V=1", "install"
   end
+
+  test do
+    expected_checksum = "4b4c2dce4c055f0a2adb93d571987a3d40c96c6cbfd9244d19b9708ce5aea454"
```

Comment:
We can safely assume this won't change. The last change to this tarball was at the start of 2006.